src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+// INITIAL STATE
+
+export const initialState = [];
+
+// BOOK REDUCER
+
+const bookReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'ADD':
+      return [...state, action.book];
+    case 'DELETE':
+      return state.filter((book) => book.id !== action.id);
+    default:
+      return state;
+  }
+};
+
+// action for adding a book
+
+export const addBook = (book) => (
+  {
+    type: 'ADD',
+    book,
+  }
+);
+
+// action for removing a book
+
+export const deleteBook = (id) => (
+  {
+    type: 'DELETE',
+    id,
+  }
+);
+
+export default bookReducer;
```

Comment:
- [ ] Please note that the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: const ADD_BOOK = bookstore/books/ADD_BOOK and const REMOVE_BOOK = bookstore/books/REMOVE_BOOK. Fix it in order to follow the project requirement :+1: 

Check the example below :arrow_down: 
```
const ADD = 'BOOKSTORE/books/ADD';
const REMOVE = 'BOOKSTORE/books/REMOVE';

export default (state = [], action) => {
  switch (action.type) {
    case ADD:
      return [...state, {
        title: action.title,
        author: action.author,
        id: action.id,
      },
      ];

    case REMOVE:
      return state.filter((book) => book.id !== action.id);

    default:
      return state;
  }
};

export const addBook = (bookTitle, bookAuthor, bookId) => ({
  type: ADD,
  title: bookTitle,
  author: bookAuthor,
  id: bookId,
});

export const removeBook = (bookId) => ({ type: REMOVE, id: bookId });
```